client/spec/blog_settings_spec.js

```diff
@@ -1,8 +1,7 @@
 var utils = require('../app/scripts/bower_components/superdesk/client/spec/helpers/utils'),
     login = utils.login,
-    expectBlog = require('./helpers/utils').expectBlog,
-    openBlog = require('./helpers/utils').openBlog,
-    blogs = require('./helpers/utils').blogs;
+    randomString = require('./helpers/pages').randomString,
+    list = require('./helpers/pages').list;
```

Comment:
Can we have an explicit name for this ? I don't understand what is a `list` from `pages`